lib/active_merchant/billing/gateways/securion_pay.rb

```diff
@@ -155,6 +164,16 @@ def add_external_three_ds(post, options)
         }
       end
 
+      # TO-DO
+      # def add_stored_credentials(post, options)
+      #   return unless options[:stored_credential]
+      #   initiator = options.dig(:stored_credential, :initiator)
+      #   _reason_type = options.dig(:stored_credential, :reason_type)
+      #   initiator = 'customer' if initiator == 'cardholder'
+      #   # first_recurring, subsequent_recurring, merchant_initiated, customer_initiated, merchant_installments
+      #   post[:type] = initiator + '_' + type
+      # end
+
```

Comment:
This block of code is marked as TO-DO, at what point could it be uncommented, could you specify in a comment? or removed from the PR :)

Reply:
This is work in progress pending on solving some doubts with shift4 staff